### ISSUE TITLE
[Headings] One vs. Many h1

### ISSUE THREAD

**Issue by yatil** [w3c/wai-tutorials#496]
@nitedog in #486:

>  also other questions, like "is it allowed to have more than one H1". These are only implicitly addressed on this page, and it may be good to be more specific (as you do with skipping levels).
> 
>  I also don't have a specific suggestion. We ran into a similar problem with the images tutorial, and ended up with a "tips and tricks" section. On the "styling" page of the carousels tutorial you also seem to have a set of generic factoids but approached differently than "tips and tricks". Any of these approaches could work, but I would suggest to do this as directly as possible, like FAQ style. Hope these thoughts provide some inspiration...

**Comment by yatil** [comment]
Oops, @nitedog, I accidentally added a section on One vs. Many h1 to the tutorial:

https://github.com/w3c/wai-tutorials/commit/02367ea918e78c7bb6d7a019efcf4db279bfe7c7#diff-3d2808bc3afe668dbbc36760c5b5fe68R128

I don’t know what happened!

**Comment by yatil** [comment]
(We can decide to not include it in the next publication.)

**Comment by nitedog** [comment]
hahaha - you are too quick for yourself.

I think it may be a bit too wordy. Also, may be better to collect all questions we want to address then come up with an approach for the set. It may be only one or two, not sure.

**Comment by yatil** [comment]
We didn’t hear any other comments for this page.

**Comment by rachelcarmena** [comment]
Recently, I made a talk about accessibility. I needed to talk about what I'd learnt after making a lot of mistakes, because of a lack of knowledge.
I read a lot of documentation about accessibility and I found the recommendation about "only one h1 per page" in several resources, but now I realize that W3C doesn't include it explicitly. 

Please, do you know more about this issue? Is this document updated: https://www.w3.org/wiki/HTML/Usage/Headings/h1only? 
Has the actual document outline exposed by browsers and assistive technology changed?

Now, I think it makes sense to have several h1 with HTML5. In order words, it makes sense not to worry about the order of the rest of the headings if you add a `<section>` or `<article>`.

I'll write a post about my talk very soon, so I can take that opportunity to clarify this issue.
Thanks in advance!

**Comment by yatil** [comment]
@rachelcarmena 

> Please, do you know more about this issue? Is this document updated: https://www.w3.org/wiki/HTML/Usage/Headings/h1only?
> Has the actual document outline exposed by browsers and assistive technology changed?
> 
> Now, I think it makes sense to have several h1 with HTML5. In order words, it makes sense not to worry about the order of the rest of the headings if you add a <section> or <article>.

No browser or screen reader I know of does support using H1 elements only. If you depend on the HTML5 document outline algorithm, websites are likely harder to access for many, many users.

**Comment by yatil** [comment]
@rachelcarmena BTW, the WAI IG discussion mailing list is a good place to ask such questions: https://www.w3.org/WAI/about/groups/waiig/#mailinglist

**Comment by rachelcarmena** [comment]
Oh, sorry, I've just sent my question to w3c-wai-ig@w3.org. Thank you very much, @yatil. I didn't know those mailing lists. I'm sure they will be very useful for me, in order to know more about accessibility. Thanks Eric!
